### PR TITLE
Kickoff

### PR DESCRIPTION
**Pull request by hdf1986**
# Summary
- Added rspec
- Added rubocop
- Basic gem bootstrap

# Note

Anyone else wants to write the gem description? :P

### CODE REVIEW

File: MIT-LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright 2017 hdf1986
```
Comment (mishuagopian) [inline MIT-LICENSE]: copy paste this one https://github.com/Wolox/wor-authentication/blob/master/LICENSE.md

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+```bash
+$ bundle
+```
+
+Or install it yourself as:
+```bash
+$ gem install wor-paginate
+```
+
+## Contributing
```
Comment (mishuagopian) [inline README.md]: Copy from contributing till the end https://github.com/Wolox/wor-authentication/blob/master/README.md#contributing

Comment (mishuagopian) [inline README.md]: modify maintainers and gems name in license

File: spec/spy.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#
+# Class used in tests to keep track of amount of calls for a method in a klass.
```
Comment (mishuagopian) [inline spec/spy.rb]: Loving this classsssss

File: wor-paginate.gemspec [hunk 4]
```diff
@@ -0,0 +1,33 @@
+$:.push File.expand_path("../lib", __FILE__)
+
+require "wor/paginate/version"
```
Comment (mishuagopian) [inline wor-paginate.gemspec]: I think you have to require date here too

Comment (hdf1986) [inline wor-paginate.gemspec]: Hmm are you sure? i tried building the gem without requiring date and it works fine

**Comment by hdf1986** [comment]
@mishuagopian Fixed :)

File: spec/spy.rb (hunk 3 above)
Comment (alebian) [inline spec/spy.rb]: LOL

File: README.md [hunk 5]
```diff
@@ -24,5 +24,31 @@ $ gem install wor-paginate
...
 
+## About ##
+
+This project is maintained by [Hugo Farji](https://github.com/hdf1986), [Ignacio Coluccio](https://github.com/icoluccio), and [Alan Halatian](https://github.com/alanhala). It was written by [Wolox](http://www.wolox.com.ar).
+![Wolox](https://raw.githubusercontent.com/Wolox/press-kit/master/logos/logo_banner.png)
+
 ## License
-The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+
+**wor-paginate** is available under the MIT [license](https://raw.githubusercontent.com/Wolox/wor-authentication/master/LICENSE.md).
```
Comment (mishuagopian) [inline README.md]: Lol. It says `wor-authentication`. Copy paste FTW.

File: README.md [hunk 6]
```diff
@@ -1 +1,54 @@
...
+$ bundle
+```
+
+Or install it yourself as:
+```bash
+$ gem install wor-paginate
+```
+
+## Contributing
+Contribution directions go here.
```
Comment (mishuagopian) [inline README.md]: What about the contributing section??

File: wor-paginate.gemspec (hunk 4 above)
Comment (mishuagopian) [inline wor-paginate.gemspec]: Pretty sure. You'll see it failing when building in travis

File: README.md [hunk 7]
```diff
@@ -1 +1,54 @@
...
+
+## About ##
+
+This project is maintained by [Hugo Farji](https://github.com/hdf1986), [Ignacio Coluccio](https://github.com/icoluccio), and [Alan Halatian](https://github.com/alanhala). It was written by [Wolox](http://www.wolox.com.ar).
+![Wolox](https://raw.githubusercontent.com/Wolox/press-kit/master/logos/logo_banner.png)
+
+## License
+
+**wor-paginate** is available under the MIT [license](https://raw.githubusercontent.com/Wolox/wor-authentication/master/LICENSE.md).
```
Comment (mishuagopian) [inline README.md]: `wor-authentication` copy paste ftw

File: wor-paginate.gemspec [hunk 8]
```diff
@@ -1,6 +1,7 @@
 $:.push File.expand_path("../lib", __FILE__)
 
 require "wor/paginate/version"
+require 'date'
```
Comment (mishuagopian) [inline wor-paginate.gemspec]: I knew it!

**Pull request merged**